testLinking_funcs.py

```diff
@@ -1,45 +1,42 @@
 import linkingFuncs
-import pytest
 
 
-class TestClass:
-    # def test_search_ticker():
-    #     result = linkingFuncs.search_tiker("121edscawdfvbe")
-    #     result2 = linkingFuncs.search_tiker("AAPL")
-    #     print(result)
-    #     print(result2)
+# def test_search_ticker():
+#     result = linkingFuncs.search_tiker("121edscawdfvbe")
+#     result2 = linkingFuncs.search_tiker("AAPL")
+#     print(result)
+#     print(result2)
 
-    # def test_viewDetail_info():
-    #     result3 = linkingFuncs.viewDetail_info("1d", "msft")
-    #     print(result3)
+# def test_viewDetail_info():
```

Comment:
Are these commented out tests going to be removed or are eventually added in?

Reply:
Consider unit tests instead maybe

Reply:
We will eventually add them in